providers/mongo/src/airflow/providers/mongo/hooks/mongo.py

```diff
@@ -225,6 +225,39 @@ def get_collection(self, mongo_collection: str, mongo_db: str | None = None) ->
 
         return mongo_conn.get_database(mongo_db).get_collection(mongo_collection)
 
+    def create_collection(
+        self,
+        mongo_collection: str,
+        mongo_db: str | None = None,
+        create_if_exists: bool = True,
```

Comment:
```suggestion
        return_if_exists: bool = True,
```

should it be `return_if_exists` instead

Reply:
good suggestion, it makes more sense